holoviews/plotting/bokeh/path.py

```diff
@@ -6,25 +6,32 @@
 from bokeh.models import HoverTool, FactorRange
 
 from ...core import util
-from .element import ElementPlot, ColorbarPlot, line_properties, fill_properties
+from .element import ColorbarPlot, line_properties, fill_properties
 from .util import expand_batched_style
 
 
-class PathPlot(ElementPlot):
+class PathPlot(ColorbarPlot):
+
+    color_index = param.ClassSelector(default=None, class_=(util.basestring, int),
+                                      allow_None=True, doc="""
+      Index of the dimension from which the color will the drawn""")
```

Comment:
Would be nice to get the ``op`` proposal in soon...

Reply:
Agreed.